### PR TITLE
fix(statutory): fix setting fee paid for members list. Fixes HELP-1305

### PR DESCRIPTION
**Pull request by serge1peshcoff**
apparently we send `fee_paid` as string and for whatever reason backend also returns it as string (it stores it properly though), so I've done 2 things: converting this value to a float and 2) updated the input so it'd only accept float values (so if you'll type `aaa`, it'll complain)

### CODE REVIEW

File: src/views/statutory/ListMembersLists.vue [hunk 1]
```diff
@@ -84,9 +84,11 @@ export default {
         message: 'Set fee paid',
         inputAttrs: {
           placeholder: 'Fee paid (in EUR)',
-          required: true
+          required: true,
+          type: 'number',
+          step: 0.001
```
Comment (WikiRik) [inline src/views/statutory/ListMembersLists.vue:89]: Why is this 0.001 instead of 0.01?

Comment (serge1peshcoff) [reply src/views/statutory/ListMembersLists.vue:89]: actually yes, I'll fix it

Comment (serge1peshcoff) [reply src/views/statutory/ListMembersLists.vue]: done

**Review by WikiRik — APPROVED** [review]

**Pull request merged**